embulk-executor-mapreduce/src/main/java/org/embulk/executor/mapreduce/MapReduceExecutor.java

```diff
@@ -384,6 +384,12 @@ public AttemptState getAttemptState()
                 builder.add(new AttemptReport(aid, state));
             } catch (EOFException ex) {  // plus Not Found exception
                 builder.add(new AttemptReport(aid, null));
+            } catch (IOException ex) {
```

Comment:
The error occurs when the corrupted file is opened (https://github.com/embulk/embulk-executor-mapreduce/blob/skip_attempt_state_file_reading_failures/embulk-executor-mapreduce/src/main/java/org/embulk/executor/mapreduce/EmbulkMapReduce.java#L164). Basically we cannot handle it on our side.